app/src/main/java/com/example/habitsmasher/UserRegistrationFragment.java

```diff
@@ -153,6 +153,9 @@ private void checkForUniqueUsername(FirestoreCallback firestoreCallback, String
      * @param username the user's username
      */
     private void createNewUserWithEmailAndPassword(String email, String password, String username) {
+        PasswordEncrypt passwordEncrypt = new PasswordEncrypt();
+        password = passwordEncrypt.encrypt(password);
+        String finalPassword = password;
```

Comment:
Instead of `finalPassword`, let's name this `encryptedPassword` instead.

- [ ] rename `finalPassword`